Add french translations

Molkobain opened the pull request.

Molkobain committed:
Add french translations

AHalfWildSheep commented:
Hi Molkobain, thanks for the contribution! 😊
I corrected a dictionary entry key and added a new one with 9a9d548b, would you mind adding them to your PR?

Regards

steffunky commented on fr.dict.ahws-dashlet-stats.php:
```diff
@@ -0,0 +1,39 @@
+	'UI:DashletStatsCompare:Label' => 'Dashlet de stats. comparatives',
+	'UI:DashletStatsCompare:Description' => 'Dashlet badge pour comparer deux fonctions d\'aggrégation',
+	'UI:DashletStatsCompare:Prop:CompareQuery' => 'Requête de comparaison',
+	'UI:DashletStatsCompare:Prop:CompareUnit' => 'Unité de comparaison del\'écart',
```
Missing space between "de" et "l'écart"

Molkobain replied:
Done !

Molkobain committed:
Fix typo

Molkobain commented:
> Hi Molkobain, thanks for the contribution! 😊
> I corrected a dictionary entry key and added a new one with [9a9d548](https://github.com/AHalfWildSheep/itop-dashlet-stats/commit/9a9d548b0b8aeb3cd5533e4897bc10c65246670b), would you mind adding them to your PR?
> 
> Regards

Sure, I update my branch soon and let know

Molkobain committed:
Merge remote-tracking branch 'upstream/master' into master

Molkobain commented:
I'm closing this PR as I messed up. Modifications were on the master branch, so the repo is now out of sync. Re-doing it on a dedicated branch.
